index.js

```diff
@@ -10,6 +10,14 @@ exports.constants = {
 
 // https://en.wikipedia.org/wiki/ANSI_escape_code#CSI_(Control_Sequence_Introducer)_sequences
 
+exports.hideCursor = function hideCursor() {
+  return CSI + '?25l';
+};
+
+exports.showCursor = function showCursor() {
+  return CSI + '?25h';
+};
```

Comment:
```suggestion
exports.hideCursor = CSI + '?25l'
exports.showCursor = CSI + '?25h'
```

These are constants so there's no need for them to be functions.